packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js

```diff
@@ -20,17 +20,16 @@ import CategoryProductList from 'Component/CategoryProductList';
 import CategorySort from 'Component/CategorySort';
 import ContentWrapper from 'Component/ContentWrapper';
 import Html from 'Component/Html';
-import Image from 'Component/Image/Image.container';
 import Loader from 'Component/Loader';
-import grid from 'Style/icons/grid.svg';
-import list from 'Style/icons/list.svg';
+import GridIcon from 'Style/icons/Grid';
+import ListIcon from 'Style/icons/List';
 import { CategoryTreeType } from 'Type/Category';
 import { DeviceType } from 'Type/Device';
 import { FilterInputType, FilterType } from 'Type/ProductList';
 import { isCrawler, isSSR } from 'Util/Browser';
 import BrowserDatabase from 'Util/BrowserDatabase';
 
-import filterIcon from '../../style/icons/filter.svg';
+import FilterIcon from '../../style/icons/Filter';
```

Comment:
Please import FilterIcon like other icons.
Relative path shouldn't be used here.